shell/browser/javascript_environment.cc

```diff
@@ -100,6 +99,11 @@ void JavascriptEnvironment::OnMessageLoopDestroying() {
   base::MessageLoopCurrent::Get()->RemoveTaskObserver(microtasks_runner_.get());
   platform_->DrainTasks(isolate_);
   platform_->UnregisterIsolate(isolate_);
+  {
+    v8::Locker locker(isolate_);
+    v8::HandleScope scope(isolate_);
+    gin_helper::CleanedUpAtExit::DoCleanup();
```

Comment:
Should this block be moved up to the top of this function, so that it gets the benefit of microtask runner and the v8::platform draining the event loop ?

Considering this block was handled in the destructor before and nothing seemed to break, it should be fine as it is,